openmetadata-ui/src/main/resources/ui/src/locale/languages/de-de.json

```diff
@@ -394,6 +394,7 @@
     "domain-lowercase-plural": "bereiche",
     "domain-plural": "Bereiche",
     "domain-type": "Bereichstyp",
+    "domain-version": "Domain version",
```

Comment:
would prefer to have a common key for this

"entity-version": "{{entity}} version",